install.md

```diff
@@ -99,33 +99,44 @@ To install Application Service Adapter:
       -extensions SAN -config <(cat /etc/ssl/openssl.cnf <(printf "[ SAN ]\nsubjectAltName='DNS:<API-FQDN>'")) \
      -days 365
     ```
-1. Create a `tas-adapter-values.yml` file with the desired installation settings, following the package setting schema.
-
-    You can use the following sample as a template:
-
-    ```yaml
-    ---
-    api_ingress:
-      fqdn: "<API-FQDN>"
-      replicas: <API-REPLICA-COUNT>
-      tls:
-        crt: |
-          <TLS-CRT>
-        key: |
-          <TLS-KEY>
-    package_registry_base_path: "<PACKAGE-REGISTRY-BASE>"
-    kpack_image_tag_prefix: "<KPACK-TAG-PREFIX>"
-    ```
-
-    Where:
-    <ul>
-      <li>`<API-FQDN>` is the FQDN that you want to use for the Application Service Adapter API.</li>
-      <li>`<API-REPLICA-COUNT>` is the desired number of instances for the Application Service Adapter API deployment.</li>
-      <li>`<TLS-CRT>` is the PEM-encoded public certificate for the Application Service Adapter API.</li>
-      <li>`<TLS-KEY>` is the PEM-encoded private key for the Application Service Adapter API.</li>
-      <li>`<PACKAGE-REGISTRY-BASE>` is the container registry "folder"/"project" where application source code (Packages) will be uploaded.</li>
-      <li>`<KPACK-TAG-PREFIX>` is the container registry "folder"/"project" where runnable application images (Droplets) will be uploaded.</li>
-    </ul>
+1. Create a `tas-adapter-values.yml` file with the desired installation settings, following the schema specified for the package.
+
+   The following values are required:
+
+   ```yaml
+   ---
+   api_ingress:
+     fqdn: "<API-FQDN>"
+     tls:
+       crt: "<TLS-CRT>"
+       key: "<TLS-KEY>"
+   package_registry_base_path: "<PACKAGE-REGISTRY-BASE>"
+   kpack_image_tag_prefix: "<KPACK-TAG-PREFIX>"
+   ```
+
+   Where:
+   * `<API-FQDN>` is the FQDN that you want to use for the TAS adapter API.
+   * `<API-REPLICA-COUNT>` is the desired number of instances for the TAS adapter API deployment.
+   * `<TLS-CRT>` is the PEM-encoded public certificate for the TAS adapter API.
+   * `<TLS-KEY>` is the PEM-encoded private key for the TAS adapter API.
+   * `<PACKAGE-REGISTRY-BASE>` is the container registry "folder"/"project" where application source code (Packages) will be uploaded
+   * `<KPACK-TAG-PREFIX>` is the container registry "folder"/"project" where runnable application images (Droplets) will be uploaded
```

Comment:
Could you change this back to use `<ul>` and `<li>` tags, and remove the `API-REPLICA-COUNT` line? Whatever Markdown renderer VMware docs uses doesn't process the markdown UL syntax correctly.

Reply:
(That is, it doesn't process it correctly when nested like this.)

Reply:
Interesting - we were wondering about why there was inline html.

Reply:
@emalm has anyone tried it with '-' instead of '*' to see if that will work better in the VMware docs? It seems better to me if we can avoid weird mashups of html and md, if at all possible.

Reply:
@julian-hj I haven't tried that, but it's worth a shot later today. I too would prefer we use Markdown as much as possible instead of dropping down to "raw" HTML!